### DISCUSSION TITLE
VScode notebook never stops executing but issues no errors

### DISCUSSION THREAD

**Discussion by zstern** [microsoft/vscode-jupyter#13031] [Questions and Answers]
When executing a cell the cell just keeps running and never stops and never issues an error.  VSCODE juputer notebooks used to work so I rolled back the install a bit but still get the same problem.  It seems to be mostly on imports.  I can run something simple like print('hello') which does execute but then when I go to import anything even native packages it just shows the loading errors and line animation across the top but nothing ever concludes.  Even if i restart the kernal the notebook acts like its still executing.  

I'm using vscode Version: 1.76.0 (Universal)
Commit: 92da9481c0904c6adfe372c12da3b7748d74bdcb

and jupyter extension v2023.1.2000312134.  On a macbook pro with m1 chip.  I don't know what changed as the notebook extension used to work and now it does not.

Thanks,

**Comment by DonJayamanne** [comment]
> so I rolled back the install a bit but still get the same problem. 

sounds like something else has changed outside vscode. Else it shoudl work in the pervious version.

Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `verbose`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel.

**Reply by zstern** [reply]
So maybe I hadn't restarted vscode after rolling back (thought I did) but now things seem to be working EXCEPT the small blue line that animates across the top of the cell while executing never stops even after the cell has executed.  It can keep going on all the cells that get executed even though they have completed the operation.  

I did turn on verbose logging in the settings for jupyter but where should I be seeing the logs at? You say 'output panel' do you mean the results of executing the cell? Nothing shows up there (except the outcome of the cell as expected).  If i do something like print('hello world') it prints and finishes as it should. If i try something like assigning a variable test = 'asdf' it will show the small blue line animation indefinitely.  Thank you!
